framework/test/mocha/integration/rounds.js

```diff
@@ -837,8 +832,8 @@ describe('rounds', () => {
 
 			it('delegates list should be equal to one generated at the beginning of round 1', async () => {
 				const lastBlock = library.modules.blocks.lastBlock;
-				return library.modules.rounds
-					.generateDelegateList(slots.calcRound(lastBlock.height + 1), null)
+				return library.modules.dpos
+					.getRoundDelegates(slots.calcRound(lastBlock.height + 1))
```

Comment:
same goes here and the lines below.. for awaiting?

Reply:
Done